packages/generator/src/batch/type.ts

```diff
@@ -30,12 +33,16 @@ export function writeRequestType(
 }
 
 function getWriteRequestType(service: VdmServiceMetadata): string {
-  return service.entities
-    .map(
-      e =>
-        `CreateRequestBuilder<${e.className}<DeSerializersT>, DeSerializersT> | UpdateRequestBuilder<${e.className}<DeSerializersT>, DeSerializersT> | DeleteRequestBuilder<${e.className}<DeSerializersT>, DeSerializersT>`
-    )
-    .join(' | ');
+  const entites = service.entities.map(
```

Comment:
I prefer to have a name for each part by using variable, because the difference for exampe in the `GetAllRequestBuilder` and `GetByKeyRequestBuilder` is hard to spot.

Reply:
however, consistentcy is more important than personal preference so I align the two cases.